doc/nrf/doc_styleguide.rst

```diff
@@ -257,34 +337,37 @@ To link to functions, enums, or structs from within Doxygen itself, use the
 .. code-block:: c
    :caption: Reference documentation example
 
-	/** @brief Event header structure.
-	 *  Use this structure with the function @ref function_name and
-         *  this structure is related to another structure, @ref structure_name.
-	 */
+    /** @brief Event header structure.
+     *  Use this structure with the function @ref function_name and
+     *  this structure is related to another structure, @ref structure_name.
+     */
 
 .. note::
-   Linking to functions does not currently work due to `breathe issue #438`_.
+   Linking to functions does not currently work due to `Breathe issue #438`_.
```

Comment:
Do you know if this is still true, @ru-fu ?

Reply:
The issue is still open, so I don't expect it to be fixed.
(This is for linking to a functions within doxygen, not from RST to doxygen.)